### PR TITLE
Feature/emails

### PR DESCRIPTION
**Pull request by adrianovalente**
**This PR:**
- [x] Adds column `email` to `user`
- [x] Updates `/signIn` to accept user email instead of CPF
- [x] Changes `postgres.js` to rebuild the database whenever the project is deployed in production (actually it's not a good practice but that all we can do untill #22 is fixed)

Related issue: #23

### CODE REVIEW

**Comment by adrianovalente** [comment]
@StefanoHellebrekers take a look here

File: models/users.js [hunk 1]
```diff
@@ -24,11 +24,13 @@ function findUserByPasswordAndCPF(cpf, password, cb) {
 }
 
 function insertUser(user, cb) {
-  const values = [user.cpf, user.password, user.rg, user.phone, user.name, user.type || 'client'];
+  const values = [user.cpf, user.password, user.rg, user.email, user.phone, user.name, user.type || 'client'];
```
Comment (lucasBgabriel) [inline models/users.js]: Can you pls use parenthesis??

like:
```
const values = [(user.cpf, user.password, user.rg, user.email, user.phone, user.name, user.type) || ('client')];

File: models/users.js [hunk 2]
```diff
@@ -24,11 +24,13 @@ function findUserByPasswordAndCPF(cpf, password, cb) {
...
 
   db.insert('users', values, function(err) {
     if(!err) return cb();
-    if(!!err.detail && err.detail.indexOf('already exists') > -1) return cb(errors.CPF_ALREADY_USED);
+    if(!!err.detail && err.detail.indexOf('already exists') > -1) {
```
Comment (lucasBgabriel) [inline models/users.js]: can you pls remove this magic number ( -1)???

**Comment by StefanoHellebrekers** [comment]
:+1:

File: models/users.js (hunk 1 above)
Comment (adrianovalente) [inline models/users.js]: @lucasBgabriel actually it should be
```javascript
const values = [user.cpf, user.password, user.rg, user.email, user.phone, user.name, (user.type || 'client')]
```

**Comment by lucasBgabriel** [comment]
👍

**Pull request merged**